core/src/test/java/org/infinispan/marshall/core/StoreAsBinaryTest.java

```diff
@@ -412,7 +411,8 @@ public void testEntrySetValueFromEntryCollections() {
       m.put(key3, value3);
       cache(0, "replSync").putAll(m);
 
-      Set<Map.Entry<Object, Object>> entries = cache(0, "replSync").entrySet();
+      Cache<Object, Object> replSync = cache(0, "replSync");
```

Comment:
Why add the extra variable?

Reply:
I liked the extra variable, there's a lot of repetition in this test :)